src/views/dataset/MultiSourceConfiguration.vue

```diff
@@ -555,19 +557,69 @@ export default class MultiSourceConfiguration extends Vue {
       );
     }
 
+    // Check for OIB files; for whatever reason, these need to be
+    // explicitly turned into large images before we can get information from it
+    const fileExtensions = names.map((name: string) => {
+      const parts = name.split(".");
+      return parts.length > 1 ? parts[parts.length - 1].toLowerCase() : "";
+    });
+    const hasOibFiles = fileExtensions.includes("oib");
+
+    // For OIB files, explicitly try to create a large image first
+    if (hasOibFiles) {
+      try {
+        // Process each OIB file to ensure it has a large image
+        for (let i = 0; i < items.length; i++) {
+          const item = items[i];
```

Comment:
you probably can do `for (const item of items) { ...}` then you can remove `const item = items[i];`

Reply:
Thanks!